main.nf

```diff
@@ -40,22 +40,13 @@ WorkflowMain.initialise(workflow, params, log)
 ========================================================================================
 */
 
-workflow NFCORE_RNASEQ {
-
-    //
-    // WORKFLOW: Get SRA run information for public database ids, download and md5sum check FastQ files, auto-create samplesheet
-    //
-    if (params.public_data_ids) {
-        include { SRA_DOWNLOAD } from './workflows/sra_download'
-        SRA_DOWNLOAD ()
+include { RNASEQ } from './workflows/rnaseq'
```

Comment:
It became a bug now see [here](https://github.com/nextflow-io/nextflow/commit/7dbf64bea38907126f44b09a023b8061bf3363d0) :open_mouth:

Reply:
Yup. Saw that! Fixed, just in the nick of time 😆